examples/embedded_resource_manager_ex.py

```diff
@@ -0,0 +1,20 @@
+import sys, os
+sys.path.append(os.getcwd())
+from locust_plugins.embedded_resource_manager import HttpUserWithResources
+from locust import task, events
+import time
+
+class TestUserWithResources(HttpUserWithResources):
+    bundle_resource_stats=False
+    default_resource_filter=".*[^(js)]$"
+    @task
+    def include_resources_true(self):
+        response = self.client.get("/", resource_filter=".*[^(js)]$")
+
+    @task
+    def include_resources_missing(self):
```

Comment:
"missing"?

Reply:
Oh, did you just mean that the parameter is missing? how about "default" instead? or just "include_resources". Put this one first as well.

Reply:
Will do